packages/js-core/src/website/lib/sync.ts

```diff
@@ -27,7 +26,7 @@ const syncWithBackend = async (
     }
 
     // if user id is not available
-    const url = baseUrl + urlSuffix;
+    const url = baseUrl;
```

Comment:
no need for redeclaring this variable, pls just use the `baseUrl` below this line, or rename `baseUrl` to just `url` 🙂